.github/workflows/Build.yaml

```diff
@@ -173,22 +207,30 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
 
-      - name: Build projects before running emulator
+      - name: Accept Android licenses
+        run: yes | "$ANDROID_HOME"/cmdline-tools/latest/bin/sdkmanager --licenses || true
+
+      - name: Build AndroidTest apps
         run: ./gradlew packageDemoDebug packageDemoDebugAndroidTest
 
-      - name: Run instrumentation tests
-        uses: reactivecircus/android-emulator-runner@v2
+      - name: Run instrumented tests with GMD
+        run: ./gradlew ciDemoDebugAndroidTest --no-parallel --max-workers=1
```

Comment:
You're using ` --no-parallel` and `max-worker == 1`. Isn't the `parallel` redundant here?